pages/docs/reference/opt-in-requirements.md

```diff
@@ -287,12 +287,12 @@ fun getTime(): Time {}
 </div>
 
 
-## Opt-in requirements for experimental APIs
+## Opt-in requirements for pre-stable APIs
```

Comment:
I think "experimental" is more correct here though since it also describes code of external users, for whom "experimental" might be a more recognizable term. Or do we want to encourage the use of the term "pre-stable" for our users too?

Reply:
We certainly want to avoid confusion and double-meanings for terms. I'll look for some other options

Reply:
Having read the text belov, the first line says:

> If you use opt-in requirements for features that are not stable yet, carefully handle the API graduation to avoid 
breaking the client code.

Looks like pre-stable is fine